client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/InteractionEventForm.tsx

```diff
@@ -1,260 +1,238 @@
-                    </Collapse>
-                }
-                <Grid className={formClasses.formRow} container justify='flex-start'>
-                    <Grid item xs={3}>
-                        <FormInput fieldName='משעה'>
-                            <div className={classes.contactDatePicker}>
-                                <TimePick
-                                    required
-                                    test-id='contactLocationStartTime'
-                                    labelText='משעה'
-                                    value={startTime}
-                                    onChange={(newTime: Date) => handleTimeChange(newTime, interactionEventDialogData.startTime, 'startTime')}
-                                />
-                            </div>
-                        </FormInput>
-                    </Grid>
-                    <Grid item xs={3} className={classes.contactDate}>
-                        <FormInput fieldName='עד שעה'>
-                            <div className={classes.contactDatePicker}>
-                                <TimePick
-                                    required
-                                    test-id='contactLocationEndTime'
-                                    labelText='עד שעה'
-                                    value={endTime}
-                                    onChange={(newTime: Date) => handleTimeChange(newTime, interactionEventDialogData.endTime, 'endTime')}
-                                />
-                            </div>
-                        </FormInput>
-                    </Grid>
-                </Grid>
-                <Grid item xs={6}>
-                    <FormInput fieldName='האם מותר להחצנה'>
-                        <Toggle
-                            test-id='allowExternalization'
-                            className={formClasses.formToggle}
-                            value={externalizationApproval}
-                            onChange={onExternalizationApprovalChange}/>
-                    </FormInput>
-                </Grid>
+  }
+
+  const onSubmit = (data: InteractionEventDialogData) => {
+    saveIntreactions(data)
+  }
+
+  return (
+    <>
+      <FormProvider {...methods}>
+      <form id="interactionEventForm" onSubmit={methods.handleSubmit(onSubmit)}>
+        <Grid className={formClasses.form} container justify="flex-start">
+          <PlacesTypesAndSubTypes
+            control={methods.control}
+            placeTypeName={InteractionEventDialogFields.PLACE_TYPE}
+            placeSubTypeName={InteractionEventDialogFields.PLACE_SUB_TYPE}
+            placeType={placeType}
+            placeSubType={placeSubType}
+            onPlaceTypeChange={(newValue)=> methods.setValue(InteractionEventDialogFields.PLACE_TYPE, newValue)}
+            onPlaceSubTypeChange={(newValue)=> methods.setValue(InteractionEventDialogFields.PLACE_SUB_TYPE, newValue)}
+          />
+          {placeType === privateHouse.code && (
```

Comment:
Don't need the conditionm if the result is false the collapse won't show the content